lib/src/main/kotlin/com/dotypos/lib/migration/dto/entity/markeeta/LicenseMarkeetaMigrationDto.kt

```diff
@@ -80,6 +80,12 @@ data class LicenseMarkeetaMigrationDto (
     @SerialName("commissionPartner")
     val commissionPartner: CommissionPartner?,
 
+    /**
+     * Wholesale Customer.
+     */
+    @SerialName("wholesaleCustomer")
```

Comment:
I would expect WholesaleCustomer entity. Maybe wholesaleCustomerName would be better.